pkg/security/security.go

```diff
@@ -500,6 +502,11 @@ func GetIstioSDSServerSocketPath() string {
 	return filepath.Join(WorkloadIdentityPath, DefaultWorkloadIdentitySocketFile)
 }
 
+func GetIstioSDSFileServerSocketPath() string {
+	// TODO: wrong path!!!
+	return filepath.Join(WorkloadIdentityCredentialsPath, "files-socket")
```

Comment:
I vote to reuse `CredentialNameSocketPath` and put 2x sockets in the same dir there.

Reply:
Good call